packages/fauna/index.js

```diff
@@ -0,0 +1,537 @@
+import { query as q } from 'faunadb'
```

Comment:
I think this file should be transpiled to something(?), since it is now an individual package consumed by other projects. @williamluke4 do you have a suggestion? Could we use `tsc` as a `build` script to do this, since it is used already for the Prisma Adapter? In that case, does it need to be installed in `devDependencies` for all packages, or can we install it for all at the same time with `lerna` somehow?